src/fetch.js

```diff
@@ -6,9 +6,7 @@ import AWS from 'aws-sdk'
 
 // setup aws credentials
 gu.init({www:false});
-var credentials = new AWS.SharedIniFileCredentials(gu.config.aws_profile);
```

Comment:
Can we also remove https://github.com/guardian/gudocs/blob/100953047029dfbae5175e8f3c7c0da1a431e226/gu.json#L11?